hathor/event/resources/event.py

```diff
@@ -0,0 +1,146 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from itertools import islice
+from typing import List, Optional
+
+from pydantic import Field, NonNegativeInt
+
+from hathor.api_util import Resource, set_cors
+from hathor.cli.openapi_files.register import register_resource
+from hathor.conf import HathorSettings
+from hathor.event import BaseEvent
+from hathor.manager import HathorManager
+from hathor.utils.api import ErrorResponse, QueryParams, Response
+
+settings = HathorSettings()
+
+
+@register_resource
+class EventResource(Resource):
+    isLeaf = True
+
+    def __init__(self, manager: HathorManager):
```

Comment:
It seems you don't need the `HathorManager` object. Why don't you inject the `EventManager` directly?

Reply:
Makes sense! I've changed it.

Reply:
Thanks!